packages/apollo-vscode/src/extension.ts

```diff
@@ -74,7 +84,8 @@ export function activate(context: ExtensionContext) {
       fileEvents: [
         workspace.createFileSystemWatcher("**/apollo.config.js"),
         workspace.createFileSystemWatcher("**/package.json"),
-        workspace.createFileSystemWatcher("**/*.{graphql,js,ts,jsx,tsx}")
+        workspace.createFileSystemWatcher("**/*.{graphql,js,ts,jsx,tsx}"),
```

Comment:
Even though `.graphql` is the standard, I bump into a lot of cases where people use `.gql`. Perhaps it would be worthwhile to support both?

Reply:
This is out of context for the PR, would you mind submitting an issue or PR for it? Thanks!